--- unsimplified
+++ simplified
@@ -18,7 +18,7 @@
   ): Node
 
   """Reads and enables pagination through a set of `GoalContributor`."""
-  allGoalContributors(
+  goalContributors(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -47,7 +47,7 @@
   ): GoalContributorsConnection
 
   """Reads a set of `GoalContributor`."""
-  allGoalContributorsList(
+  goalContributorsList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -64,7 +64,7 @@
   ): [GoalContributor!]
 
   """Reads and enables pagination through a set of `Goal`."""
-  allGoals(
+  goals(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -93,7 +93,7 @@
   ): GoalsConnection
 
   """Reads a set of `Goal`."""
-  allGoalsList(
+  goalsList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -110,7 +110,7 @@
   ): [Goal!]
 
   """Reads and enables pagination through a set of `Organization`."""
-  allOrganizations(
+  organizations(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -139,7 +139,7 @@
   ): OrganizationsConnection
 
   """Reads a set of `Organization`."""
-  allOrganizationsList(
+  organizationsList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -156,7 +156,7 @@
   ): [Organization!]
 
   """Reads and enables pagination through a set of `Team`."""
-  allTeams(
+  teams(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -185,7 +185,7 @@
   ): TeamsConnection
 
   """Reads a set of `Team`."""
-  allTeamsList(
+  teamsList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -202,7 +202,7 @@
   ): [Team!]
 
   """Reads and enables pagination through a set of `User`."""
-  allUsers(
+  users(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -231,7 +231,7 @@
   ): UsersConnection
 
   """Reads a set of `User`."""
-  allUsersList(
+  usersList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -246,14 +246,14 @@
     """
     condition: UserCondition
   ): [User!]
-  goalContributorById(id: Int!): GoalContributor
-  goalByOrganizationIdAndTeamIdAndGoalUuid(organizationId: Int!, teamId: Int!, goalUuid: UUID!): Goal
-  organizationById(id: Int!): Organization
-  teamById(id: Int!): Team
-  userById(id: Int!): User
+  goalContributor(id: Int!): GoalContributor
+  goal(organizationId: Int!, teamId: Int!, goalUuid: UUID!): Goal
+  organization(id: Int!): Organization
+  team(id: Int!): Team
+  user(id: Int!): User
 
   """Reads a single `GoalContributor` using its globally unique `ID`."""
-  goalContributor(
+  goalContributorByNodeId(
     """
     The globally unique `ID` to be used in selecting a single `GoalContributor`.
     """
@@ -261,13 +261,13 @@
   ): GoalContributor
 
   """Reads a single `Goal` using its globally unique `ID`."""
-  goal(
+  goalByNodeId(
     """The globally unique `ID` to be used in selecting a single `Goal`."""
     nodeId: ID!
   ): Goal
 
   """Reads a single `Organization` using its globally unique `ID`."""
-  organization(
+  organizationByNodeId(
     """
     The globally unique `ID` to be used in selecting a single `Organization`.
     """
@@ -275,13 +275,13 @@
   ): Organization
 
   """Reads a single `Team` using its globally unique `ID`."""
-  team(
+  teamByNodeId(
     """The globally unique `ID` to be used in selecting a single `Team`."""
     nodeId: ID!
   ): Team
 
   """Reads a single `User` using its globally unique `ID`."""
-  user(
+  userByNodeId(
     """The globally unique `ID` to be used in selecting a single `User`."""
     nodeId: ID!
   ): User
@@ -329,16 +329,16 @@
   """
   Reads a single `Organization` that is related to this `GoalContributor`.
   """
-  organizationByOrganizationId: Organization
+  organization: Organization
 
   """Reads a single `Goal` that is related to this `GoalContributor`."""
-  goalByOrganizationIdAndTeamIdAndGoalUuid: Goal
+  organizationTeamGoal: Goal
 
   """Reads a single `Team` that is related to this `GoalContributor`."""
-  teamByTeamId: Team
+  team: Team
 
   """Reads a single `User` that is related to this `GoalContributor`."""
-  userByContributorId: User
+  contributor: User
 }
 
 """
@@ -355,7 +355,7 @@
   name: String
 
   """Reads and enables pagination through a set of `Goal`."""
-  goalsByOrganizationId(
+  goals(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -384,7 +384,7 @@
   ): GoalsConnection!
 
   """Reads and enables pagination through a set of `Goal`."""
-  goalsByOrganizationIdList(
+  goalsList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -401,7 +401,7 @@
   ): [Goal!]!
 
   """Reads and enables pagination through a set of `GoalContributor`."""
-  goalContributorsByOrganizationId(
+  goalContributors(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -430,7 +430,7 @@
   ): GoalContributorsConnection!
 
   """Reads and enables pagination through a set of `GoalContributor`."""
-  goalContributorsByOrganizationIdList(
+  goalContributorsList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -475,10 +475,10 @@
   aim: String
 
   """Reads a single `Organization` that is related to this `Goal`."""
-  organizationByOrganizationId: Organization
+  organization: Organization
 
   """Reads a single `Team` that is related to this `Goal`."""
-  teamByTeamId: Team
+  team: Team
 
   """Reads and enables pagination through a set of `GoalContributor`."""
   goalContributorsByOrganizationIdAndTeamIdAndGoalUuid(
@@ -536,7 +536,7 @@
   name: String
 
   """Reads and enables pagination through a set of `Goal`."""
-  goalsByTeamId(
+  goals(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -565,7 +565,7 @@
   ): GoalsConnection!
 
   """Reads and enables pagination through a set of `Goal`."""
-  goalsByTeamIdList(
+  goalsList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -582,7 +582,7 @@
   ): [Goal!]!
 
   """Reads and enables pagination through a set of `GoalContributor`."""
-  goalContributorsByTeamId(
+  goalContributors(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -611,7 +611,7 @@
   ): GoalContributorsConnection!
 
   """Reads and enables pagination through a set of `GoalContributor`."""
-  goalContributorsByTeamIdList(
+  goalContributorsList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -986,165 +986,165 @@
   """
   Updates a single `GoalContributor` using its globally unique id and a patch.
   """
-  updateGoalContributor(
+  updateGoalContributorByNodeId(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: UpdateGoalContributorInput!
+    input: UpdateGoalContributorByNodeIdInput!
   ): UpdateGoalContributorPayload
 
   """Updates a single `GoalContributor` using a unique key and a patch."""
-  updateGoalContributorById(
+  updateGoalContributor(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: UpdateGoalContributorByIdInput!
+    input: UpdateGoalContributorInput!
   ): UpdateGoalContributorPayload
 
   """Updates a single `Goal` using its globally unique id and a patch."""
-  updateGoal(
+  updateGoalByNodeId(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: UpdateGoalInput!
+    input: UpdateGoalByNodeIdInput!
   ): UpdateGoalPayload
 
   """Updates a single `Goal` using a unique key and a patch."""
-  updateGoalByOrganizationIdAndTeamIdAndGoalUuid(
+  updateGoal(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: UpdateGoalByOrganizationIdAndTeamIdAndGoalUuidInput!
+    input: UpdateGoalInput!
   ): UpdateGoalPayload
 
   """
   Updates a single `Organization` using its globally unique id and a patch.
   """
-  updateOrganization(
+  updateOrganizationByNodeId(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: UpdateOrganizationInput!
+    input: UpdateOrganizationByNodeIdInput!
   ): UpdateOrganizationPayload
 
   """Updates a single `Organization` using a unique key and a patch."""
-  updateOrganizationById(
+  updateOrganization(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: UpdateOrganizationByIdInput!
+    input: UpdateOrganizationInput!
   ): UpdateOrganizationPayload
 
   """Updates a single `Team` using its globally unique id and a patch."""
-  updateTeam(
+  updateTeamByNodeId(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: UpdateTeamInput!
+    input: UpdateTeamByNodeIdInput!
   ): UpdateTeamPayload
 
   """Updates a single `Team` using a unique key and a patch."""
-  updateTeamById(
+  updateTeam(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: UpdateTeamByIdInput!
+    input: UpdateTeamInput!
   ): UpdateTeamPayload
 
   """Updates a single `User` using its globally unique id and a patch."""
-  updateUser(
+  updateUserByNodeId(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: UpdateUserInput!
+    input: UpdateUserByNodeIdInput!
   ): UpdateUserPayload
 
   """Updates a single `User` using a unique key and a patch."""
-  updateUserById(
+  updateUser(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: UpdateUserByIdInput!
+    input: UpdateUserInput!
   ): UpdateUserPayload
 
   """Deletes a single `GoalContributor` using its globally unique id."""
-  deleteGoalContributor(
+  deleteGoalContributorByNodeId(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: DeleteGoalContributorInput!
+    input: DeleteGoalContributorByNodeIdInput!
   ): DeleteGoalContributorPayload
 
   """Deletes a single `GoalContributor` using a unique key."""
-  deleteGoalContributorById(
+  deleteGoalContributor(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: DeleteGoalContributorByIdInput!
+    input: DeleteGoalContributorInput!
   ): DeleteGoalContributorPayload
 
   """Deletes a single `Goal` using its globally unique id."""
-  deleteGoal(
+  deleteGoalByNodeId(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: DeleteGoalInput!
+    input: DeleteGoalByNodeIdInput!
   ): DeleteGoalPayload
 
   """Deletes a single `Goal` using a unique key."""
-  deleteGoalByOrganizationIdAndTeamIdAndGoalUuid(
+  deleteGoal(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: DeleteGoalByOrganizationIdAndTeamIdAndGoalUuidInput!
+    input: DeleteGoalInput!
   ): DeleteGoalPayload
 
   """Deletes a single `Organization` using its globally unique id."""
-  deleteOrganization(
+  deleteOrganizationByNodeId(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: DeleteOrganizationInput!
+    input: DeleteOrganizationByNodeIdInput!
   ): DeleteOrganizationPayload
 
   """Deletes a single `Organization` using a unique key."""
-  deleteOrganizationById(
+  deleteOrganization(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: DeleteOrganizationByIdInput!
+    input: DeleteOrganizationInput!
   ): DeleteOrganizationPayload
 
   """Deletes a single `Team` using its globally unique id."""
-  deleteTeam(
+  deleteTeamByNodeId(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: DeleteTeamInput!
+    input: DeleteTeamByNodeIdInput!
   ): DeleteTeamPayload
 
   """Deletes a single `Team` using a unique key."""
-  deleteTeamById(
+  deleteTeam(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: DeleteTeamByIdInput!
+    input: DeleteTeamInput!
   ): DeleteTeamPayload
 
   """Deletes a single `User` using its globally unique id."""
-  deleteUser(
+  deleteUserByNodeId(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: DeleteUserInput!
+    input: DeleteUserByNodeIdInput!
   ): DeleteUserPayload
 
   """Deletes a single `User` using a unique key."""
-  deleteUserById(
+  deleteUser(
     """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
     """
-    input: DeleteUserByIdInput!
+    input: DeleteUserInput!
   ): DeleteUserPayload
 }
 
@@ -1167,16 +1167,16 @@
   """
   Reads a single `Organization` that is related to this `GoalContributor`.
   """
-  organizationByOrganizationId: Organization
+  organization: Organization
 
   """Reads a single `Goal` that is related to this `GoalContributor`."""
-  goalByOrganizationIdAndTeamIdAndGoalUuid: Goal
+  organizationTeamGoal: Goal
 
   """Reads a single `Team` that is related to this `GoalContributor`."""
-  teamByTeamId: Team
+  team: Team
 
   """Reads a single `User` that is related to this `GoalContributor`."""
-  userByContributorId: User
+  contributor: User
 
   """An edge for our `GoalContributor`. May be used by Relay 1."""
   goalContributorEdge(
@@ -1224,10 +1224,10 @@
   query: Query
 
   """Reads a single `Organization` that is related to this `Goal`."""
-  organizationByOrganizationId: Organization
+  organization: Organization
 
   """Reads a single `Team` that is related to this `Goal`."""
-  teamByTeamId: Team
+  team: Team
 
   """An edge for our `Goal`. May be used by Relay 1."""
   goalEdge(
@@ -1398,16 +1398,16 @@
   """
   Reads a single `Organization` that is related to this `GoalContributor`.
   """
-  organizationByOrganizationId: Organization
+  organization: Organization
 
   """Reads a single `Goal` that is related to this `GoalContributor`."""
-  goalByOrganizationIdAndTeamIdAndGoalUuid: Goal
+  organizationTeamGoal: Goal
 
   """Reads a single `Team` that is related to this `GoalContributor`."""
-  teamByTeamId: Team
+  team: Team
 
   """Reads a single `User` that is related to this `GoalContributor`."""
-  userByContributorId: User
+  contributor: User
 
   """An edge for our `GoalContributor`. May be used by Relay 1."""
   goalContributorEdge(
@@ -1416,8 +1416,8 @@
   ): GoalContributorsEdge
 }
 
-"""All input for the `updateGoalContributor` mutation."""
-input UpdateGoalContributorInput {
+"""All input for the `updateGoalContributorByNodeId` mutation."""
+input UpdateGoalContributorByNodeIdInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1432,7 +1432,7 @@
   """
   An object where the defined keys will be set on the `GoalContributor` being updated.
   """
-  goalContributorPatch: GoalContributorPatch!
+  patch: GoalContributorPatch!
 }
 
 """
@@ -1447,8 +1447,8 @@
   contribution: String
 }
 
-"""All input for the `updateGoalContributorById` mutation."""
-input UpdateGoalContributorByIdInput {
+"""All input for the `updateGoalContributor` mutation."""
+input UpdateGoalContributorInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1458,7 +1458,7 @@
   """
   An object where the defined keys will be set on the `GoalContributor` being updated.
   """
-  goalContributorPatch: GoalContributorPatch!
+  patch: GoalContributorPatch!
   id: Int!
 }
 
@@ -1479,10 +1479,10 @@
   query: Query
 
   """Reads a single `Organization` that is related to this `Goal`."""
-  organizationByOrganizationId: Organization
+  organization: Organization
 
   """Reads a single `Team` that is related to this `Goal`."""
-  teamByTeamId: Team
+  team: Team
 
   """An edge for our `Goal`. May be used by Relay 1."""
   goalEdge(
@@ -1491,8 +1491,8 @@
   ): GoalsEdge
 }
 
-"""All input for the `updateGoal` mutation."""
-input UpdateGoalInput {
+"""All input for the `updateGoalByNodeId` mutation."""
+input UpdateGoalByNodeIdInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1507,7 +1507,7 @@
   """
   An object where the defined keys will be set on the `Goal` being updated.
   """
-  goalPatch: GoalPatch!
+  patch: GoalPatch!
 }
 
 """Represents an update to a `Goal`. Fields that are set will be updated."""
@@ -1518,10 +1518,8 @@
   aim: String
 }
 
-"""
-All input for the `updateGoalByOrganizationIdAndTeamIdAndGoalUuid` mutation.
-"""
-input UpdateGoalByOrganizationIdAndTeamIdAndGoalUuidInput {
+"""All input for the `updateGoal` mutation."""
+input UpdateGoalInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1531,7 +1529,7 @@
   """
   An object where the defined keys will be set on the `Goal` being updated.
   """
-  goalPatch: GoalPatch!
+  patch: GoalPatch!
   organizationId: Int!
   teamId: Int!
   goalUuid: UUID!
@@ -1560,8 +1558,8 @@
   ): OrganizationsEdge
 }
 
-"""All input for the `updateOrganization` mutation."""
-input UpdateOrganizationInput {
+"""All input for the `updateOrganizationByNodeId` mutation."""
+input UpdateOrganizationByNodeIdInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1576,7 +1574,7 @@
   """
   An object where the defined keys will be set on the `Organization` being updated.
   """
-  organizationPatch: OrganizationPatch!
+  patch: OrganizationPatch!
 }
 
 """
@@ -1587,8 +1585,8 @@
   name: String
 }
 
-"""All input for the `updateOrganizationById` mutation."""
-input UpdateOrganizationByIdInput {
+"""All input for the `updateOrganization` mutation."""
+input UpdateOrganizationInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1598,7 +1596,7 @@
   """
   An object where the defined keys will be set on the `Organization` being updated.
   """
-  organizationPatch: OrganizationPatch!
+  patch: OrganizationPatch!
   id: Int!
 }
 
@@ -1625,8 +1623,8 @@
   ): TeamsEdge
 }
 
-"""All input for the `updateTeam` mutation."""
-input UpdateTeamInput {
+"""All input for the `updateTeamByNodeId` mutation."""
+input UpdateTeamByNodeIdInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1641,7 +1639,7 @@
   """
   An object where the defined keys will be set on the `Team` being updated.
   """
-  teamPatch: TeamPatch!
+  patch: TeamPatch!
 }
 
 """Represents an update to a `Team`. Fields that are set will be updated."""
@@ -1650,8 +1648,8 @@
   name: String
 }
 
-"""All input for the `updateTeamById` mutation."""
-input UpdateTeamByIdInput {
+"""All input for the `updateTeam` mutation."""
+input UpdateTeamInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1661,7 +1659,7 @@
   """
   An object where the defined keys will be set on the `Team` being updated.
   """
-  teamPatch: TeamPatch!
+  patch: TeamPatch!
   id: Int!
 }
 
@@ -1688,8 +1686,8 @@
   ): UsersEdge
 }
 
-"""All input for the `updateUser` mutation."""
-input UpdateUserInput {
+"""All input for the `updateUserByNodeId` mutation."""
+input UpdateUserByNodeIdInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1704,7 +1702,7 @@
   """
   An object where the defined keys will be set on the `User` being updated.
   """
-  userPatch: UserPatch!
+  patch: UserPatch!
 }
 
 """Represents an update to a `User`. Fields that are set will be updated."""
@@ -1713,8 +1711,8 @@
   username: String
 }
 
-"""All input for the `updateUserById` mutation."""
-input UpdateUserByIdInput {
+"""All input for the `updateUser` mutation."""
+input UpdateUserInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1724,7 +1722,7 @@
   """
   An object where the defined keys will be set on the `User` being updated.
   """
-  userPatch: UserPatch!
+  patch: UserPatch!
   id: Int!
 }
 
@@ -1738,7 +1736,7 @@
 
   """The `GoalContributor` that was deleted by this mutation."""
   goalContributor: GoalContributor
-  deletedGoalContributorId: ID
+  deletedGoalContributorNodeId: ID
 
   """
   Our root query field type. Allows us to run any query from our mutation payload.
@@ -1748,16 +1746,16 @@
   """
   Reads a single `Organization` that is related to this `GoalContributor`.
   """
-  organizationByOrganizationId: Organization
+  organization: Organization
 
   """Reads a single `Goal` that is related to this `GoalContributor`."""
-  goalByOrganizationIdAndTeamIdAndGoalUuid: Goal
+  organizationTeamGoal: Goal
 
   """Reads a single `Team` that is related to this `GoalContributor`."""
-  teamByTeamId: Team
+  team: Team
 
   """Reads a single `User` that is related to this `GoalContributor`."""
-  userByContributorId: User
+  contributor: User
 
   """An edge for our `GoalContributor`. May be used by Relay 1."""
   goalContributorEdge(
@@ -1766,8 +1764,8 @@
   ): GoalContributorsEdge
 }
 
-"""All input for the `deleteGoalContributor` mutation."""
-input DeleteGoalContributorInput {
+"""All input for the `deleteGoalContributorByNodeId` mutation."""
+input DeleteGoalContributorByNodeIdInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1780,8 +1778,8 @@
   nodeId: ID!
 }
 
-"""All input for the `deleteGoalContributorById` mutation."""
-input DeleteGoalContributorByIdInput {
+"""All input for the `deleteGoalContributor` mutation."""
+input DeleteGoalContributorInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1800,7 +1798,7 @@
 
   """The `Goal` that was deleted by this mutation."""
   goal: Goal
-  deletedGoalId: ID
+  deletedGoalNodeId: ID
 
   """
   Our root query field type. Allows us to run any query from our mutation payload.
@@ -1808,10 +1806,10 @@
   query: Query
 
   """Reads a single `Organization` that is related to this `Goal`."""
-  organizationByOrganizationId: Organization
+  organization: Organization
 
   """Reads a single `Team` that is related to this `Goal`."""
-  teamByTeamId: Team
+  team: Team
 
   """An edge for our `Goal`. May be used by Relay 1."""
   goalEdge(
@@ -1820,8 +1818,8 @@
   ): GoalsEdge
 }
 
-"""All input for the `deleteGoal` mutation."""
-input DeleteGoalInput {
+"""All input for the `deleteGoalByNodeId` mutation."""
+input DeleteGoalByNodeIdInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1834,10 +1832,8 @@
   nodeId: ID!
 }
 
-"""
-All input for the `deleteGoalByOrganizationIdAndTeamIdAndGoalUuid` mutation.
-"""
-input DeleteGoalByOrganizationIdAndTeamIdAndGoalUuidInput {
+"""All input for the `deleteGoal` mutation."""
+input DeleteGoalInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1858,7 +1854,7 @@
 
   """The `Organization` that was deleted by this mutation."""
   organization: Organization
-  deletedOrganizationId: ID
+  deletedOrganizationNodeId: ID
 
   """
   Our root query field type. Allows us to run any query from our mutation payload.
@@ -1872,8 +1868,8 @@
   ): OrganizationsEdge
 }
 
-"""All input for the `deleteOrganization` mutation."""
-input DeleteOrganizationInput {
+"""All input for the `deleteOrganizationByNodeId` mutation."""
+input DeleteOrganizationByNodeIdInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1886,8 +1882,8 @@
   nodeId: ID!
 }
 
-"""All input for the `deleteOrganizationById` mutation."""
-input DeleteOrganizationByIdInput {
+"""All input for the `deleteOrganization` mutation."""
+input DeleteOrganizationInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1906,7 +1902,7 @@
 
   """The `Team` that was deleted by this mutation."""
   team: Team
-  deletedTeamId: ID
+  deletedTeamNodeId: ID
 
   """
   Our root query field type. Allows us to run any query from our mutation payload.
@@ -1920,8 +1916,8 @@
   ): TeamsEdge
 }
 
-"""All input for the `deleteTeam` mutation."""
-input DeleteTeamInput {
+"""All input for the `deleteTeamByNodeId` mutation."""
+input DeleteTeamByNodeIdInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1934,8 +1930,8 @@
   nodeId: ID!
 }
 
-"""All input for the `deleteTeamById` mutation."""
-input DeleteTeamByIdInput {
+"""All input for the `deleteTeam` mutation."""
+input DeleteTeamInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1954,7 +1950,7 @@
 
   """The `User` that was deleted by this mutation."""
   user: User
-  deletedUserId: ID
+  deletedUserNodeId: ID
 
   """
   Our root query field type. Allows us to run any query from our mutation payload.
@@ -1968,8 +1964,8 @@
   ): UsersEdge
 }
 
-"""All input for the `deleteUser` mutation."""
-input DeleteUserInput {
+"""All input for the `deleteUserByNodeId` mutation."""
+input DeleteUserByNodeIdInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
@@ -1982,8 +1978,8 @@
   nodeId: ID!
 }
 
-"""All input for the `deleteUserById` mutation."""
-input DeleteUserByIdInput {
+"""All input for the `deleteUser` mutation."""
+input DeleteUserInput {
   """
   An arbitrary string value with no semantic meaning. Will be included in the
   payload verbatim. May be used to track mutations by the client.
