--- unsimplified
+++ simplified
@@ -18,7 +18,7 @@
   ): Node
 
   """Reads and enables pagination through a set of `Beverage`."""
-  allBeverages(
+  beverages(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -47,7 +47,7 @@
   ): BeveragesConnection
 
   """Reads a set of `Beverage`."""
-  allBeveragesList(
+  beveragesList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -64,7 +64,7 @@
   ): [Beverage!]
 
   """Reads and enables pagination through a set of `Company`."""
-  allCompanies(
+  companiesConnection(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -93,7 +93,7 @@
   ): CompaniesConnection
 
   """Reads a set of `Company`."""
-  allCompaniesList(
+  companies(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -108,8 +108,8 @@
     """
     condition: CompanyCondition
   ): [Company!]
-  beverageById(id: Int!): Beverage
-  companyById(id: Int!): Company
+  beverage(id: Int!): Beverage
+  company(id: Int!): Company
 
   """Reads and enables pagination through a set of `Beverage`."""
   listInclude(
@@ -142,7 +142,7 @@
   ): [Beverage]
 
   """Reads and enables pagination through a set of `Beverage`."""
-  listOmit(
+  listOmitConnection(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -163,7 +163,7 @@
   ): BeveragesConnection
 
   """Reads and enables pagination through a set of `Beverage`."""
-  listOmitList(
+  listOmit(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -172,13 +172,13 @@
   ): [Beverage]
 
   """Reads a single `Beverage` using its globally unique `ID`."""
-  beverage(
+  beverageByNodeId(
     """The globally unique `ID` to be used in selecting a single `Beverage`."""
     nodeId: ID!
   ): Beverage
 
   """Reads a single `Company` using its globally unique `ID`."""
-  company(
+  companyByNodeId(
     """The globally unique `ID` to be used in selecting a single `Company`."""
     nodeId: ID!
   ): Company
@@ -220,10 +220,10 @@
   name: String!
 
   """Reads a single `Company` that is related to this `Beverage`."""
-  companyByCompanyId: Company
+  company: Company
 
   """Reads a single `Company` that is related to this `Beverage`."""
-  companyByDistributorId: Company
+  distributor: Company
 }
 
 type Company implements Node {
@@ -235,7 +235,7 @@
   name: String!
 
   """Reads and enables pagination through a set of `Beverage`."""
-  beveragesByCompanyId(
+  beveragesConnection(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -264,7 +264,7 @@
   ): BeveragesConnection!
 
   """Reads and enables pagination through a set of `Beverage`."""
-  beveragesByCompanyIdList(
+  beverages(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -357,7 +357,7 @@
   ): [Beverage]
 
   """Reads and enables pagination through a set of `Beverage`."""
-  computedListOmit(
+  computedListOmitConnection(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -378,7 +378,7 @@
   ): BeveragesConnection!
 
   """Reads and enables pagination through a set of `Beverage`."""
-  computedListOmitList(
+  computedListOmit(
     """Only read the first `n` values of the set."""
     first: Int
 
