--- unsimplified
+++ simplified
@@ -9,10 +9,10 @@
   name: String!
 
   """Reads a single `Company` that is related to this `Beverage`."""
-  companyByCompanyId: Company
+  company: Company
 
   """Reads a single `Company` that is related to this `Beverage`."""
-  companyByDistributorId: Company
+  distributor: Company
 }
 
 """
@@ -120,7 +120,7 @@
   name: String!
 
   """Reads and enables pagination through a set of `Beverage`."""
-  beveragesByCompanyId(
+  beveragesConnection(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -149,7 +149,7 @@
   ): BeveragesConnection!
 
   """Reads and enables pagination through a set of `Beverage`."""
-  beveragesByCompanyIdList(
+  beverages(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -242,7 +242,7 @@
   ): [Beverage]
 
   """Reads and enables pagination through a set of `Beverage`."""
-  computedListOmit(
+  computedListOmitConnection(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -263,7 +263,7 @@
   ): BeveragesConnection!
 
   """Reads and enables pagination through a set of `Beverage`."""
-  computedListOmitList(
+  computedListOmit(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -329,7 +329,7 @@
   ): Node
 
   """Reads and enables pagination through a set of `Beverage`."""
-  allBeverages(
+  beverages(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -358,7 +358,7 @@
   ): BeveragesConnection
 
   """Reads a set of `Beverage`."""
-  allBeveragesList(
+  beveragesList(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -375,7 +375,7 @@
   ): [Beverage!]
 
   """Reads and enables pagination through a set of `Company`."""
-  allCompanies(
+  companiesConnection(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -404,7 +404,7 @@
   ): CompaniesConnection
 
   """Reads a set of `Company`."""
-  allCompaniesList(
+  companies(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -419,8 +419,8 @@
     """
     condition: CompanyCondition
   ): [Company!]
-  beverageById(id: Int!): Beverage
-  companyById(id: Int!): Company
+  beverage(id: Int!): Beverage
+  company(id: Int!): Company
 
   """Reads and enables pagination through a set of `Beverage`."""
   listInclude(
@@ -453,7 +453,7 @@
   ): [Beverage]
 
   """Reads and enables pagination through a set of `Beverage`."""
-  listOmit(
+  listOmitConnection(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -474,7 +474,7 @@
   ): BeveragesConnection
 
   """Reads and enables pagination through a set of `Beverage`."""
-  listOmitList(
+  listOmit(
     """Only read the first `n` values of the set."""
     first: Int
 
@@ -483,13 +483,13 @@
   ): [Beverage]
 
   """Reads a single `Beverage` using its globally unique `ID`."""
-  beverage(
+  beverageByNodeId(
     """The globally unique `ID` to be used in selecting a single `Beverage`."""
     nodeId: ID!
   ): Beverage
 
   """Reads a single `Company` using its globally unique `ID`."""
-  company(
+  companyByNodeId(
     """The globally unique `ID` to be used in selecting a single `Company`."""
     nodeId: ID!
   ): Company
